scripts-available/CDB_Overviews.sql

```diff
@@ -904,14 +808,20 @@ AS $$
       columns := columns || ', n AS _feature_count';
     END IF;
 
-    EXECUTE Format('DROP TABLE IF EXISTS %I.%I CASCADE;', schema_name, overview_rel);
+    overview_table_name := Format('%I.%I', schema_name, overview_rel);
```

Comment:
Agreed, it should be refactored. I also agree we can leave it out for a latter ticket.